intro-html.md

```diff
@@ -130,4 +130,13 @@ Has te zril voluptatum philosophia, cum velit graeci mnesarchum id, ex his nulla
 ```
 
 ## Workshop
-TODO
\ No newline at end of file
+TODO
+
+####Tribute Page
```

Comment:
I can try to do the same without Bootstrap. Want me to go ahead and do that? I can host it and send you a link on codepen.io